server/src/main/scala/test/TestDrtSystem.scala

```diff
@@ -38,6 +46,21 @@ class TestDrtSystem(override val actorSystem: ActorSystem, override val config:
   override lazy val voyageManifestsStage: Source[ManifestsFeedResponse, NotUsed] = voyageManifestTestSourceGraph
   val testFeed = TestFixtureFeed(system)
 
+
+  sys.env.get("LIVE_FIXTURE_CSV").foreach { file =>
```

Comment:
Should we rather add config to application.conf and reference it using `config`?

Reply:
because this is only in Test I didn't want to pollute the application.conf (to keep the seam as mall as possible with test stuff) but happy to do so if you think it's more consistent.